transformer_engine/common/layer_norm/ln_bwd_kernels.cuh

```diff
@@ -103,6 +103,9 @@ void ln_bwd_tuned_kernel(layer_norm::BwdParams params) {
                 compute_t x_tmp = x[it].data.elt[jt];
                 compute_t y_tmp = rs_r * (x_tmp - mu_r);
                 compute_t dy_tmp = compute_t(gamma[it].data.elt[jt]);
+                if (params.zero_centered_gamma) {
+                  dy_tmp += static_cast<compute_t>(1);
+                }
```

Comment:
These lines (even for `zero_centered_gamma=False`) are causing the numerical changes.